Packs/CybleEventsV2/Integrations/CybleEventsV2/CybleEventsV2.py

```diff
@@ -866,6 +858,67 @@ def get_alert_by_id(client, alert_id, token, url):
         raise DemistoException(f"[get_alert_by_id] Error during HTTP request: {str(e)}")
 
 
+def update_alert_data_command(client: Client, url: str, token: str, args: Dict[str, Any]) -> CommandResults:
```

Comment:
since you're allowing updating more than one alert in one command then the name should look something like this:
cyble-vision-update-alerts